src/Adapter/Shipment/CommandHandler/MergeProductsToShipmentHandler.php

```diff
@@ -59,10 +60,18 @@ public function handle(MergeProductsToShipment $command)
             throw new ShipmentNotFoundException(sprintf('Shipment with id "%s" was not found', $sourceId));
         }
 
+        if (!empty($sourceShipment->getTrackingNumber())) {
+            throw new CannotEditShipmentShippedException(sprintf('Cannot merge the shipment "%s" because is already shipped', $sourceId));
```

Comment:
Did you make a wording review of the Exceptions with @gericfo ?

I would have said "because it is already shipped" for instance here